src/layout_abstract_syntax.rs

```diff
@@ -35,6 +35,16 @@ impl LayoutAbstractTag {
 	pub fn add_attribute(&mut self, attribute: Attribute) {
 		self.attributes.push(attribute);
 	}
+
+	/// Retrieves the `width` attribute of this node, if any.
+	pub fn width(&self) -> Option<&AttributeValue> {
+		self.attributes.iter().find(|attribute| attribute.name == "width").map(|attribute| &attribute.value)
+	}
+
+	/// Retrieves the `height` attribute of this node, if any.
+	pub fn height(&self) -> Option<&AttributeValue> {
+		self.attributes.iter().find(|attribute| attribute.name == "height").map(|attributes| &attributes.value)
+	}
```

Comment:
I've created these helper functions, but it makes sense to eventually just store the layout attributes as a dedicated structure in the node (because these are some built-in attributes).